modules/combined/test/src/base/CombinedTestApp.C

```diff
@@ -47,6 +47,8 @@ InputParameters
 CombinedTestApp::validParams()
 {
   InputParameters params = CombinedApp::validParams();
+  // Below parameter is set to enable data driven mesh generation in test suite
+  params.set<bool>(MeshGeneratorSystem::allow_data_driven_param) = true;
```

Comment:
this isnt mentioned in the documentation. We generally dont do this to accomodate one-offs.

Reply:
ok seems this is already in moose. oh well.

Reply:
I don't think this is needed anymore, is it? I don't see you adding any tests to combined

Reply:
This was needed for the Combined tests to pass, since they call all the Reactor module data driven tests

Reply:
Okay, fine with me.